src/components/ParticipantCard.spec.js

```diff
@@ -0,0 +1,14 @@
+import {render, screen} from '@testing-library/react';
+
+import ParticipantCard from './ParticipantCard.js';
+
+describe('ParticipantCardn', () => {
+  it('should render the data when api responds', () => {
+    render(<ParticipantCard />);
+    const participantCard =screen.getByRole('list');
+    expect(participantCard).toBeInTheDocument();
+    document.querySelector('section');
```

Comment:
querySelector is not needed at this point.

Reply:
okay thank u! I delete the code line.